client/signup/wpcom-login-form/index.jsx

```diff
@@ -12,46 +28,27 @@ export default class WpcomLoginForm extends Component {
 		this.form.submit();
 	}
 
-	action() {
-		const subdomainRegExp = /^https?:\/\/([a-z0-9]*).wordpress.com/;
-		let subdomain = '';
-
-		if (
-			subdomainRegExp.test( this.props.redirectTo ) &&
-			config( 'hostname' ) !== 'wpcalypso.wordpress.com' &&
-			config( 'hostname' ) !== 'horizon.wordpress.com'
-		) {
-			subdomain = this.props.redirectTo.match( subdomainRegExp )[ 1 ] + '.';
-		}
-
-		return `https://${ subdomain }wordpress.com/wp-login.php`;
-	}
-
 	renderExtraFields() {
 		const { extraFields } = this.props;
 
 		if ( ! extraFields ) {
 			return null;
 		}
 
-		return (
-			<div>
-				{ Object.keys( extraFields ).map( ( field ) => {
-					return (
-						<input key={ field } type="hidden" name={ field } value={ extraFields[ field ] } />
-					);
-				} ) }
-			</div>
-		);
+		return Object.keys( extraFields ).map( ( field ) => (
+			<input key={ field } type="hidden" name={ field } value={ extraFields[ field ] } />
+		) );
 	}
 
 	storeFormRef = ( form ) => {
 		this.form = form;
```

Comment:
Could be converted to modern ref as class instance property like `form = createRef()`

Reply:
This should be fixed via https://github.com/Automattic/wp-calypso/pull/68099/commits/55794aaad5325384f52add1c9e70a7118e14260b